talentprotocol/web-dapp
# feat: delete account confirmation flow

## Summary

This PR changes the delete account flow with a confirmation flow.

**Settings tab**

![Screenshot 2022-08-22 at 09 14 56](https://user-images.githubusercontent.com/33064059/185987846-f0ac2c6a-33f8-46b2-80ae-0041847fb20c.png)

**Email**

![Screenshot 2022-08-22 at 19 06 54](https://user-images.githubusercontent.com/33064059/185989229-e0ac1345-55d2-4b92-b9cf-e7a52fa25d74.png)


## Notion link

https://talentprotocol.notion.site/Change-delete-account-flow-e8e0adc39e394bcd92283cbd553d5379

## How to test?

- Go to "Settings"
- Scroll down to "Close Account" and click "Send delete account confirmation email"
- Open confirmation email
- Click "Close my Talent Protocol account" CTA
- Account should be deleted and redirected to the home page (login page)
